templates/phpstan/bootstrap.php

```diff
@@ -18,8 +20,14 @@
 require_once $rootDir . '/config/bootstrap.php';
 
 // Make sure loader php-parser is coming from php stan composer
+
+// 1- Use with Docker container
+$loader = new \Composer\Autoload\ClassLoader();
+$loader->setPsr4('PhpParser\\', ['/composer/vendor/nikic/php-parser/lib/PhpParser']);
+$loader->register(true);
+// 2- Use with PHPStan phar
 $loader = new \Composer\Autoload\ClassLoader();
-$loader->setPsr4('PhpParser\\', array('/composer/vendor/nikic/php-parser/lib/PhpParser'));
+$loader->setPsr4('PhpParser\\', ['phar://' . $pathToModuleRoot . 'vendor/phpstan/phpstan-shim/phpstan.phar/vendor/nikic/php-parser/lib/PhpParser/']);
```

Comment:
this is smart :)

Reply:
Yeah, but I just remember that I had to remove this from the dependencies. It won't work anymore.